dist/index.js.cache.js

```diff
@@ -262,20 +262,88 @@ module.exports = act_on_update_command
+    try {
+        // get the issue
+        const issue = await Issue.getInstance();
+
+        if (github.context.eventName === "issue" && github.context.payload.action === "opened" && !issue.isPullRequest()) {
+            // fetch the issue labels
+            const issue_labels = issue.details.labels;
+            const hasPendingTriageLabel = issue_labels.some(label => label.name === 'pending-triage');
+            const hasMaxAssigneesLabel = issue_labels.some(label => label.name.match(/max-assignees-[0-9]{1,2}/));
+            // prepare the missing labels list
+            let missingLabels = [];
+            if (!hasPendingTriageLabel) {
+                core.info("Issue does not have any label");
+                missingLabels.push('pending-triage');
+            }
+            if (!hasMaxAssigneesLabel) {
+                core.info("Issue does not have max-assignees label");
+                // adding max-assignees-1 by default
+                missingLabels.push('max-assignees-1');
+            }
+            // add the missing labels
+            const add_labels_reponse = await octokit.rest.issues.addLabels({
+                owner: github.context.payload.repository.owner.login,
+                repo: github.context.payload.repository.name,
+                issue_number: issue.actions_payload.number,
+                labels: missingLabels,
+            });
+
+            if (add_labels_reponse.status == 200) {
+                core.info("Labels added successfully");
+            } else {
+                core.setFailed("Failed to add labels");
+                return;
+            }
+
+            // lock the conversation to author and org members unless approved by triage team
+            const lockResponse = await octokit.rest.issues.lock(
+                {
+                    owner: github.context.payload.repository.owner.login,
+                    repo: github.context.payload.repository.name,
+                    issue_number: issue.actions_payload.number,
+                }
+            );
+            if (lockResponse.status == 204) {
+                core.info("Issue locked successfully");
+            } else {
+                core.setFailed("Failed to lock issue");
+            }
+        }
+    } catch (error) {
+        core.setFailed(error?.message);
+    }
+}
+
+module.exports = add_labels_and_lock_new_issues
+
+/***/ }),
+
 /***/ 693:
 /***/ ((module, __unused_webpack_exports, __webpack_require__) => {
 
 const core = __webpack_require__(2186);
 const github = __webpack_require__(5438);
 
 // This function will verify action has been performed by a member of triage team
-function verifyTriageTeam(){
+function verifyTriageTeam() {
     const triageTeamUsernames = core.getInput("triage-team-usernames").split(",");
     const actor = github.context.actor;
     return triageTeamUsernames.includes(actor);
 }
 
-module.exports = { verifyTriageTeam };
+module.exports = { verifyTriageTeam, addLabels };
```

Comment:
@looneyD-ROHIT 
The function doesn't exist anymore. So remove and build thee package
```suggestion
module.exports = { verifyTriageTeam };
```